src/Jank/PlayerSelection/ui.tsx

```diff
@@ -0,0 +1,45 @@
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+export const ErrorText = styled.span`
+  color: red;
+  font-weight: bold;
+`;
+
+export const DescriptionText = styled.span`
+  font-weight: bold;
+  font-size: 32px;
+  margin: 20px;
+`;
+
+export const PlayerGrid = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  width: 90%;
+`;
+export const Picture = styled.img`
+border-radius: 5%;
+margin-bottom: 50px;
+width: 100%;
```

Comment:
Something's off with the indentation here and in `Card` below. When running `npm run lint` the linter will also complain:

![image](https://user-images.githubusercontent.com/481294/119530184-1247e880-bd83-11eb-8fb1-b8a4cd04d68f.png)

Reply:
I can't reproduce the problem nor do I understand what I should do to fix it.

Reply:
It's just about the code style, the CSS styles should be indented by two spaces, otherwise it's hard to tell the JS apart from the CSS.
So instead of:
```
export const Picture = styled.img`
border-radius: 5%;
margin-bottom: 50px;
width: 100%;
`;
```
it should be
```
export const Picture = styled.img`
  border-radius: 5%;
  margin-bottom: 50px;
  width: 100%;
`;
```

This should be auto-formatted with in VSCode with the right extensions, but I don't know right now which ones you would need. I guess at least `vscode-styled-components` and possibly prettier or something related.